### ISSUE TITLE
turn off EM to not estimate

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [ss3sim/ss3sim#300]
Allow ss3sim to act just as an operating model, where users can choose if they want to pass an EM or not. If no, then data are generated and sampled from returning a dat file but no estimation is performed.

**Comment by k-doering-NOAA** [comment]
Another idea @latreesedenson-NOAA and I thought of yesterday  related to this one is separating the OM modifications, sampling, and EM running in `ss3sim_base` into separate functions so that they can be called independently by users if need be.

For instance, @latreesedenson-NOAA needs to add in some environmental data before running the EM, so it would be helpful to first just get the om and sample, then run custom code to add in the environmental data, and finally run the EM (with bias adjustment).

**Comment by ChristineStawitz-NOAA** [comment]
+ 1, I was just looking for how to turn off estimation for the ICES WGGRAFY group @Kotkot and I are using ss3sim for

**Comment by kellijohnson-NOAA** [comment]
Thanks all for supporting this enhancement. Until I get to writing some code, I think that we could easily pass "no est" or something like that to the EM to speed things up, alternatively without any code additions you could just change almost all parameters to have a negative phase in the EM so almost nothing is estimated which would speed things up. I am not sure if you can actually turn them all off without leading to some errors and warnings.

**Comment by Kotkot** [comment]
For the ICES WGGRAFY work that @ChristineStawitz-NOAA mentioned above, I did some quick patch to turn off estimation in ss3sim run. It might not be the best way to go but I simply added a "skip_em" argument in the "ss3sim_base" function (see image below). Which is then added to the wrapper function such as 
 out <- run_ss3sim(iterations = iterations, simdf=df, skip_em=TRUE)

![image](https://user-images.githubusercontent.com/4493227/135146074-13710282-41cd-4910-9516-f043cf7ed265.png)

This might not be the most elegant way but it was enough for what we needed.
I can make a pull request if you think it is worth pursuing this route.
